config.go

```diff
@@ -1077,6 +1089,19 @@ type OCSPConfig struct {
 	ResponderOverrides map[string]string
 }
 
+// CertificateEventData contains contextual information for
+// an obtained, renewed or revoked certificate.
+type CertificateEventData struct {
+	// Domain or subject name of the certificate.
+	Name string
```

Comment:
```suggestion
	Names []string
```

It's true there is only 1 name on managed certificates, but certificates in general can have multiple subjects.

Reply:
Hmm, I guess, but all the emissions currently are done in the context of a single name.

Reply:
Managed certificates aren't multi-SAN, but any manually-provided certificate can have multiple SANs. We don't want to limit events to only managed certificates (for example, think events for OCSP staples, etc).

Reply:
I don't know that we can guarantee we'd never have certificate-oriented events for unmanaged certificates though. I do feel like []string is crucial here.

Reply:
What if we called it `ManagedCertificateEventData` or something?